_posts/2022-06-30-My first-take-on-KMM-Kotlin-Multiplatform-Mobile.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: My first take on KMM - Kotlin Multiplatform Mobile
+author: michal.guspiel, denzilferreira
+date: 2022-06-30 16:00:00 +0200
+excerpt: As for a first project and practice assignment in Solita I've had explored new technology - KMM and created Dev Notary, a simple application that lets users create, modify and share notes between other users.
+
+tags:
+  - KMM
+  - Android
+  - Kotlin
+  - Mobile
+  - Multiplatform
+  - iOS
+  - Swift
+  - Dev Academy
+---
+
+# Preface
+
+In order to finish the Dev Academy, I've had to develop an application as my practice assignment. I've been provided a mentor - Denzil Ferreira, whom I was guided by throughout the process. The purpose of practice assignment is to provide to a freshly recruited junior developers a safe environment, in which they can work on a small feature of an application or simple application of their own. Moreover, thanks to mentor guidance development gets done in a very proffessional fashion and is a great way to kickoff start for a new software designer.
```

Comment:
... to provide to freshly ...

Moreover, thanks to the mentor's guidance, development gets done carefully? Sounds strange to have "very professional" - there's a typo there